ref/build.sh

```diff
@@ -151,5 +151,7 @@ make -j4 VERBOSE=1
 export OMP_NUM_THREADS=4
 export OMP_PLACES=cores
 export OMP_PROC_BIND=close
+ulimit -s 12000000 
```

Comment:
This won't help because it isn't propagated to all the MPI ranks during mpi launch.  This setting has to go into the user's shell startup files (.bashrc, .cshrc, .profile, etc).

Reply:
InterpFactor set to 3, tests are now passing.